indexer/indexer_tests/trie_test.cpp

```diff
@@ -83,74 +84,82 @@ string DebugPrint(KeyValuePair const & p)
 
 struct KeyValuePairBackInserter
 {
-  vector<KeyValuePair> m_v;
   template <class TString>
-  void operator()(TString const & s, trie::FixedSizeValueReader<4>::ValueType const & rawValue)
+  void operator()(TString const & s, uint32_t const & value)
   {
-    uint32_t value;
-    memcpy(&value, &rawValue, 4);
     m_v.push_back(KeyValuePair(s, value));
   }
+
+  vector<KeyValuePair> m_v;
 };
 
-struct MaxValueCalc
+// The SingleValueSerializer and ValueList classes are similar to
+// those in indexer/string_file_values.hpp but that file
+// is not included to avoid coding_tests's dependency from indexer.
```

Comment:
All tests should be declared after all libraries in omim.pro. Normally all users of a library should not depend on tests of a library. For example, pugixml tests are platform dependent and use platform dependent features while pugixml core is portable. It's OK to move coding_tests below indexer in omim.pro